tests/test_analysis/test_activation_count.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+# Modified from
+# https://github.com/facebookresearch/fvcore/blob/main/tests/test_activation_count.py
+# pyre-ignore-all-errors[2]
+import typing
+import unittest
+from collections import Counter, defaultdict
+from typing import Any, Dict, List, Tuple
+
+import torch
+import torch.nn as nn
+from numpy import prod
+
+from mmengine.analysis import ActivationAnalyzer, activation_count
+from mmengine.analysis.jit_handles import Handle
+
+
+class SmallConvNet(nn.Module):
+    """A network with three conv layers.
+
+    This is used for testing convolution layers for activation count.
+    """
+
+    def __init__(self, input_dim: int) -> None:
+        super().__init__()
+        conv_dim1 = 8
+        conv_dim2 = 4
+        conv_dim3 = 2
+        self.conv1 = nn.Conv2d(input_dim, conv_dim1, 1, 1)
+        self.conv2 = nn.Conv2d(conv_dim1, conv_dim2, 1, 2)
+        self.conv3 = nn.Conv2d(conv_dim2, conv_dim3, 1, 2)
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x = self.conv1(x)
+        x = self.conv2(x)
+        x = self.conv3(x)
+        return x
+
+    def get_gt_activation(self, x: torch.Tensor) -> Tuple[int, int, int]:
+        x = self.conv1(x)
+        count1 = prod(list(x.size()))
+        x = self.conv2(x)
+        count2 = prod(list(x.size()))
+        x = self.conv3(x)
+        count3 = prod(list(x.size()))
+        return (count1, count2, count3)
+
+
+class TestActivationAnalyzer(unittest.TestCase):
+    """Unittest for activation_count."""
+
+    def setUp(self) -> None:
+        # nn.Linear uses a different operator based on version, so make sure
+        # we are testing the right thing.
+        lin = nn.Linear(10, 10)
+        lin_x: torch.Tensor = torch.randn(10, 10)
+        trace = torch.jit.trace(lin, (lin_x, ))
+        node_kinds = [node.kind() for node in trace.graph.nodes()]
+        assert 'aten::addmm' in node_kinds or 'aten::linear' in node_kinds
+        if 'aten::addmm' in node_kinds:
+            self.lin_op = 'addmm'
+        else:
+            self.lin_op = 'linear'
```

Comment:
`self.lin_op` is only used by `test_linear` so the former can be merged into the latter.

Reply:
op is denoted as  'addmm' in lower version of PyTorch, while 'linear' in latest version of PyTorch